tracer/test/test-applications/security/aspnet/Samples.Security.AspNetMvc5/Controllers/RenderController.cs

```diff
@@ -0,0 +1,19 @@
+using System.Text;
+using System.Web.Mvc;
+
+namespace Samples.Security.AspNetCore5.Controllers
+{
+    public class RenderController : Controller
+    {
+        public ActionResult Index(string str)
```

Comment:
This is part of the merging of the weblogs?

Reply:
no, just that the core samples have it but not the mvc one